examples/axum-multipart/Cargo.toml

```diff
@@ -4,12 +4,12 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-axum = { version = "0.7", features = ["multipart"] }
+axum = { version = "0.8.0", features = ["multipart"] }
```

Comment:
I think leaving this at 0.8 instead of 0.8.0 is better. As far as I'm aware axum doesn't break on patch increases and there's already 0.8.1 out. This is not only for this case, but all axum versions in this PR.

Reply:
`0.8` and `0.8.0` are equivalent (see https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html)

Reply:
Yeah, as the default resolution style is with the ^ which indicates it allows versions between `0.8.0 - 0.8.x`. If it was declared with = it would explicitly state the specific version. Though above, this is just matter of style, I deem to remove the unnecessary zeros but the effect is same.

Reply:
THX, TIL.
Either way, it probably should already be the newest one.

Reply:
0.8.1 was only a small readme change, so probably not worth the extra constraint :)

Reply:
Yup, if you do a clean build with it, it will and should pull the latest available version from crates :slightly_smiling_face: